cordova-lib/src/cordova/platform.js

```diff
@@ -68,6 +68,13 @@ function addHelper(cmd, hooksRunner, projectRoot, targets, opts) {
         }
     }
 
+    if (opts.usegit) {
+        msg = '\r\nWARNING: The --usegit flag has been deprecated ! \r\n' +
+              'instead, please use: `cordova platform add git-url#custom-branch`. \r\n' +
+              'e.g: cordova platform add https://github.com/apache/cordova-android.git#2.4.0 \r\n';
+        events.emit('log', msg);
```

Comment:
Ok, seems backtick is used all over the place, so don't worry about that one :smile:.